package.json

```diff
@@ -1,22 +1,22 @@
-{
-  "name": "template-fullstack",
-  "version": "1.0.0",
-  "description": "",
-  "main": "index.js",
-  "scripts": {
-    "setup": "npm i && husky install && npm --prefix ./frontend i && npm --prefix ./backend i",
-    "migrate": "cd backend/ && node migrate.js && cd ..",
-    "dev": "concurrently -n front,back -c green,yellow -t \"HH:mm:ss\" -p \"{name} {time}\" \"npm --prefix ./frontend run dev\" \"npm --prefix ./backend run dev\"",
-    "dev-front": "npm --prefix ./frontend run dev",
-    "dev-back": "npm --prefix ./backend run dev",
-    "lint": "npm --prefix ./frontend run lint && npm --prefix ./backend run lint",
-    "fix": "npm --prefix ./frontend run fix && npm --prefix ./backend run fix"
-  },
-  "keywords": [],
-  "author": "Wild Code School",
-  "license": "ISC",
-  "devDependencies": {
-    "concurrently": "^7.0.0",
-    "husky": "^7.0.0"
-  }
-}
+{
```

Comment:
This file should not update 100% of it's lines on every PR, could you please investigate a bit ?